Example/Chargebee/AppDelegate.swift

```diff
@@ -19,8 +19,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Override point for customization after application
         
         // Configure the Chargebee site and Api Key
-        Chargebee.configure(site: "test-ashwin1-test",
-                            publishableApiKey: "test_1PDU9iynvhEcPMgWAJ0QZw90d2Aw92ah")  // pass in SDK Key if available
+        Chargebee.configure(site: "hbcompany-test.chargebee.com",
```

Comment:
Remove this site id and key with placeholders <Site Id> and <Publishable API keys>

Reply:
Done.